TabloidCLI/Repositories/BlogRepository.cs

```diff
@@ -87,12 +102,30 @@ public void Delete(int id)
 
         public Blog Get(int i)
```

Comment:
could we change this variable to id to match the other methods? I know it's not part of the code you changed, but I think of "i" as "index" rather than "id"

Reply:
👍